x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/constants.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ENGINES_TITLE = i18n.translate(
```

Comment:
suggestion: Would be good to move these strings to the components where they're used. We've moved from having a constants file with strings to having these strings in the components themselves. We found that having the strings colocated with the code was more valuable than the very limited reusability we got out of these constants files.

Reply:
> Looks like you copied a few old practices from older parts of the codebase. Copying the code is great! But we have newer/better practices now that we should update this approach to.

Is there any example code/link/discussion that I can refer to, for following newer coding practices? I think it would help me for improvements

> The name of the PR should be changed to something like [Enterprise Search] Add engines list to content

Done

Reply:
In general anything under `enterprise_search_content` that is not crawler-related will be up-to-date with the latest practices. eg:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/new_index.tsx
https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/index_view_logic.ts